packages/salesforcedx-sobjects-faux-generator/src/generator/configUtil.ts

```diff
@@ -5,52 +5,28 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {
-  ConfigAggregator,
-  ConfigFile,
-  ConfigValue,
-  StateAggregator
-} from '@salesforce/core';
-import * as path from 'path';
+import { ConfigAggregator, OrgConfigProperties } from '@salesforce/core';
 
-const defaultUserNameKey = 'defaultusername';
+async function getConfigAggregator(
+  projectPath: string
+): Promise<ConfigAggregator> {
+  const origCurrentWorkingDirectory = process.cwd();
```

Comment:
Discussed as a group; this is just a quirk of the VS Code reality we find ourselves in. More details [here](https://salesforce-internal.slack.com/archives/G0213NY8PMH/p1658784269185469).